db/seeds.rb

```diff
@@ -218,14 +234,16 @@
 end_time_1 = Time.at(79200).utc.strftime("%H:%M:%S")
 start_time_2 = Time.at(39600).utc.strftime("%H:%M:%S")
 end_time_2 = Time.at(75600).utc.strftime("%H:%M:%S")
-# Seed deals for amazingcoders8mc@gmail.com
 
+# Seed deals for amazingcoders8mc@gmail.com
+activate_date = '20-09-2015'.to_datetime.in_time_zone("Singapore") - 8.hours
```

Comment:
This is because by default it will be set to UTC but our database is in Singapore. So if i convert it to singapore timing it will be 08:00 hrs therefore I minus 8 hours to make it midnight. It is not done for the rest as for the rest even when defining in UTC format it is still within the day itself. activate_date is the only one that needs to be accurate up to the hour  as wishes to views chart is in hours